tests/end-to-end/test_able_to_sent_to_commands.py

```diff
@@ -0,0 +1,187 @@
+import time
+import json
+import random
+
+import pytest
+
+import paho.mqtt.client as mqtt
+from pymodbus.client import ModbusTcpClient
+from pymodbus.payload import BinaryPayloadDecoder
+from pymodbus.constants import Endian
+
+
+@pytest.mark.end_to_end
+def test_able_to_update_coil():
+    random_value = random.randint(1, 2) % 2 == 0
+
+    message_dictionary = {"action": "testCoil", "value": random_value}
+    message = json.dumps(message_dictionary)
+
+    mqttClient = mqtt.Client()
```

Comment:
Ideally all connection and closing them should happen on set up and tear down time

Reply:
It would be nice if there was a way to clear all existing coil/register values before running the tests ... not likely to cause issues in CI because the modbus server is started fresh, and the address locations don't change, but could help to avoid issues when testing locally. I doubt there is a simple way to do that though